lib/metasploit/framework/common_engine.rb

```diff
@@ -43,6 +43,11 @@ module Metasploit::Framework::CommonEngine
     if ActiveRecord.respond_to?(:legacy_connection_handling=)
       ActiveRecord.legacy_connection_handling = false
     end
+
+    # @see https://github.com/rapid7/metasploit_data_models/blob/54a17149d5ccd0830db742d14c4987b48399ceb7/lib/metasploit_data_models/yaml.rb#L10
+    # @see https://github.com/rapid7/metasploit_data_models/blob/54a17149d5ccd0830db742d14c4987b48399ceb7/lib/metasploit_data_models/base64_serializer.rb#L28-L31
+    ActiveRecord.yaml_column_permitted_classes += MetasploitDataModels::YAML::PERMITTED_CLASSES
```

Comment:
A minor nit-pick, consider:

```suggestion
    ActiveRecord.yaml_column_permitted_classes = (ActiveRecord.yaml_column_permitted_classes + MetasploitDataModels::YAML::PERMITTED_CLASSES).uniq
```

Just to avoid duplication as the existing pattern results in two reference to `Symbol` in the array:

```
>> ActiveRecord.yaml_column_permitted_classes
=>
[Symbol,
 Range,
 Set,
 Symbol,
 Time,
 :"WEBrick::Cookie",
 :"ActionController::Parameters",
 :"ActiveModel::Attribute::FromDatabase",
 :"ActiveModel::Attribute::FromUser",
 :"ActiveModel::Attribute::WithCastValue",
 :"ActiveModel::Type::Boolean",
 :"ActiveModel::Type::Integer",
 :"ActiveModel::Type::String",
 :"ActiveRecord::Coders::JSON",
 :"ActiveSupport::TimeWithZone",
 :"ActiveSupport::TimeZone",
 :"ActiveRecord::Type::Serialized",
 :"ActiveRecord::Type::Text",
 :"ActiveSupport::HashWithIndifferentAccess",
 :"Mdm::Workspace"]
```